static/scss/popup.scss

```diff
@@ -55,6 +55,10 @@
   }
 }
 
+.downshift {
+  margin-top: 50px !important;
```

Comment:
any way we can avoid the `!important` here?

Reply:
Doesn't seem to be effective without it

Reply:
yeah, I was wondering if there is a way you could refactor it so that it wouldn't be necessary. Sometimes `!important` is necessary to get a style to stick because there is already a more-specific selector styling that element - in that case, we should specify a more specific selector to apply this style, not override the style cascase with `!important`. The issue here, I think, is that `.login-popup` already has a `margin-top` applied to it - if this selector is changed to `.login-popup.downshift` instead of just `.downshift` is should override `.login-popup` automatically. I think if you move it into the `.login-popup` block by doing:

```scss
.login-popup {
  &.downshift {
  }
}
```

you should be able to remove `!important`. This is probably good practice anyway, since currently the `.downshift` selector is not focused to the `login-popup` at all.

Reply:
Tried this, doesn't work.

Reply:
that's odd, I tried it locally and it works on my machine. maybe it's a browser specific thing? I'm using Chromium.

Reply:
I'm using Chrome too, no luck here.  Would also require changing ```<div className={`popup login-popup ${className || ""}`}>``` in `ReplyButton` to use a period instead of a space between `login-popup` and the specified `className` if any, not sure I like that idea.

Reply:
No, you shouldn't need to do that. The `className` prop is set as an HTML attribute on the DOM node that ReactDOM eventually produces, you don't need to put any periods in here.

I think if you add the `&.downshift` to the existing CSS block for `.login-popup` and make sure that the `&.downshift` block is located after the pre-existing declaration for `margin-top` it should work. Here's the diff of what I did which worked for me:

```diff
diff --git a/static/scss/popup.scss b/static/scss/popup.scss
index 2b46f5ae..dc5e1515 100644
--- a/static/scss/popup.scss
+++ b/static/scss/popup.scss
@@ -55,10 +55,6 @@
   }
 }
 
-.downshift {
-  margin-top: 50px !important;
-}
-
 .share-popup {
   .triangle {
     right: 50%;
@@ -71,6 +67,10 @@
   margin-top: 10px;
   min-width: 255px;
 
+  &.downshift {
+    margin-top: 50px;
+  }
+
   .close-popup {
     position: absolute;
     top: 0px;
```

Reply:
OK, I interpreted the `.login-popup.downshift` too literally.  Just made the change.